### ISSUE TITLE
Shift Bots event logic out of slackclient

### ISSUE THREAD

**Issue by luke-john** [fendersperth/fendersbot#12]
ie

events.js

``` js
import client from 'lib/client';
import message from 'message';

client.on('message', message);
```

message.js

``` js
import client from 'lib/client';

export default (message) => {
  const channel = client.getChannelGroupOrDMByID(message.channel);
  // do stuff
}
```

**Comment by kevincharm** [comment]
Yeah it would be good to modularise this. Perhaps still call the slack-client instance `slack` or maybe `slackClient` to be less ambiguous?

**Comment by luke-john** [comment]
Yep, client would not be a useful name.

Is there going to be any other interface with slack? If not just `slack` seems fine.

**Comment by karlbright** [comment]
This will be fixed by #23

**Comment by karlbright** [comment]
Fixed by Slack Application changes.
